commands/flags.go

```diff
@@ -0,0 +1,88 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/gruntwork-io/health-checker/options"
+	"github.com/gruntwork-io/gruntwork-cli/logging"
+	"github.com/urfave/cli"
+	"github.com/sirupsen/logrus"
+)
+
+const DEFAULT_LISTENER_IP_ADDRESS = "0.0.0.0"
+const DEFAULT_LISTENER_PORT = 5500
+
+var portFlag = cli.IntSliceFlag{
+	Name: "port",
+	Usage: fmt.Sprintf("[Required] The port number on which a TCP connection will be attempted. Specify one or more times. Example: 8000"),
+}
+
+var listenerFlag = cli.StringFlag{
+	Name:  "listener",
+	Usage: fmt.Sprintf("[Optional] The IP address and port on which inbound HTTP connections will be accepted."),
+	Value: fmt.Sprintf("%s:%d", DEFAULT_LISTENER_IP_ADDRESS, DEFAULT_LISTENER_PORT),
+}
+
+var logLevelFlag = cli.StringFlag{
+	Name:  "log-level",
+	Usage: fmt.Sprintf("[Optional] Set the log level to `LEVEL`. Must be one of: %v", logrus.AllLevels),
+	Value: logrus.InfoLevel.String(),
+}
+
+var defaultFlags = []cli.Flag{
+	portFlag,
+	listenerFlag,
+	logLevelFlag,
+}
+
+// Return true if all no options at all were passed to the CLI
+func allCliOptionsEmpty(cliContext *cli.Context) bool {
+	return cliContext.NumFlags() == 0
```

Comment:
What about NumArgs (i.e., positional args)?

Reply:
Added comments to clarify.